src/style/themes/base/components/input-presentation.js

```diff
@@ -0,0 +1,55 @@
+export default ({ palette, blackWithOpacity }) => ({
```

Comment:
I think that component specific data should be placed in the folder of that component. It could be difficult to maintain later, when we have it spread in multiple places.